python/main_loop.py

```diff
@@ -9,18 +9,24 @@
 import Path
 import time
 import matplotlib.pyplot as plt
+from sailbot_msg.msg import globalWind
 
 # Constants
 MAIN_LOOP_PERIOD_SECONDS = 0.5
 SMALL_TURN_DEGREES = 10
 LOGGED_LATLONS_PER_LINE = 3
+LOW_WIND_LOOPS_PER_PATH_UPDATE = 20
```

Comment:
While in lowWindConditions, this is the minimum number of loops until the local path can be updated for optimisation reasons.

Reply:
It's difficult to say without real world data, can you add this to our testing plan on Confluence? But good work in making it easy to change.

Reply:
Added the "Pathfinding: tune `LOW_WIND_LOOPS_PER_PATH_UPDATE`" test item for the [2022-07-16 Wet Testing Plan](https://ubcsailbot.atlassian.net/wiki/spaces/ADA2/pages/1728446482/2022-07-16+Wet+Testing+Plan).